distributor/distributor.go

```diff
@@ -0,0 +1,13 @@
+package distributor
+
+import cm "scootdev/scoot/clustermembership"
+
+/*
+ * Interface for choosing a node in a cluster to send
+ * work to.  When DistributeWork is called a node from
+ * the passed in cluster should be returned, where work
+ * can be scheduled.
+ */
+type Distributor interface {
+	DistributeWork(cluster cm.Cluster) cm.Node
```

Comment:
Yes, I totally missed that and realized it last night, since the early toy ones don't rely on work.  Added to review.